content/en/docs/concepts/configuration/taint-and-toleration.md

```diff
@@ -229,7 +229,7 @@ certain condition is true. The following taints are built in:
     as unusable. After a controller from the cloud-controller-manager initializes
     this node, the kubelet removes this taint.
 
-In version 1.13, the `TaintBasedEvictions` feature is promoted to beta and enabled by default, hence the taints are automatically
+In version 1.18, the `TaintBasedEvictions` feature is promoted to GA and enabled by default, hence the taints are automatically
 added by the NodeController (or kubelet) and the normal logic for evicting pods from nodes
 based on the Ready NodeCondition is disabled.
```

Comment:
Consider rewording this to suit a graduated feature: imagine that Kubernetes v1.23 is out and this text is in place; does it look out of place?

Reply:
@sftim do you mean remove the specific reference to 1.18? I think it makes sense to mention what version it was GA'd in for future reference, so maybe something like `As of version 1.18, ...`?

Reply:
How about adding a feature-state shortcode on line 204 or thereabouts?

`{{< feature-state for_k8s_version="1.18" state="stable" >}}`

Reply:
Try it! Add it and you'll see the result in the local preview.
or there's an example at https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state

Reply:
Done